windows/power_operation_scripts/power_operation_scripts.yml

```diff
@@ -47,13 +52,19 @@
           with_nested:
             - ['default', 'custom']
             - "{{ power_script_ops }}"
-
       rescue:
-        - name: "Collect config file of VMware Tools"
+        - name: "Collect file {{ win_power_script_log_path }}"
```

Comment:
I am afraid this variable's value won't be printed in the task name. Can you test it?

Reply:
Sure. The testing is on-going ...

Reply:
Done to test it.

Reply:
You can not use `{{ win_power_script_log_path }}` here from your testing result. Please update the task name to  `Collect VMware Tools config file and logs`